docs/README.md

```diff
@@ -0,0 +1,19 @@
+Welcome to the official Linter docs.
+
+# Examples
+
+- [Indie Linter Provider v2](examples/indie-linter-v2.md) (push-style)
+- [Standard Linter Provider v2](examples/standard-linter-v2.md) (pull-style)
+- [UI Provider v1](examples/ui-provider-v1.md)
+
+# Types
+
+- [Linter Message v2](types/linter-message-v2.md)
+- [Indie Linter Provider v2](types/indie-linter-v2.md) (push-style)
+- [Standard Linter Provider v2](types/standard-linter-v2.md) (pull-style)
+- [UI Provider v1](types/ui-provider-v1.md)
+
+# Guides
```

Comment:
"Upgrade Guides", unless you have ideas for other guides that don't fit in the provider examples?